<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsDialog</class>
 <widget class="QDialog" name="GeneralSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="settingsList">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="userInterface">
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Map Magnification:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="magnification">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Icon Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="iconSize">
           <property name="minimum">
            <number>16</number>
           </property>
           <property name="maximum">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="defaultIconSize">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="popup">
            <property name="text">
             <string>Popup Menus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="overlay">
            <property name="text">
             <string>Overlay Menus</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>36</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="routing">
      <attribute name="title">
       <string>Routing</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxMapRotation">
         <property name="text">
          <string>Rotate Map automatically</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>136</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logging">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxLogging">
         <property name="toolTip">
          <string>The GPS position is logged to a file for later use.</string>
         </property>
         <property name="whatsThis">
          <string>The GPS position is logged to a file for later use.</string>
         </property>
         <property name="text">
          <string>Enable Logging</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEditPathLogging"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonPathLogging">
         <property name="toolTip">
          <string>Click to select the logfile folder</string>
         </property>
         <property name="whatsThis">
          <string>Click to select the logfile folder</string>
         </property>
         <property name="text">
          <string>â€¦</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonClearTracklog">
         <property name="toolTip">
          <string>Click to clear the tracklog</string>
         </property>
         <property name="whatsThis">
          <string>Click to clear the tracklog</string>
         </property>
         <property name="text">
          <string>Clear Tracklog</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>78</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
